.github/workflows/pypi-release.yml

```diff
@@ -15,10 +15,10 @@ jobs:
       with:
         python-version: 3.7
     - run: python -m pip install --user --upgrade setuptools wheel kivy
-    - run: python setup.py sdist bdist_wheel
+    - run: python ${{ matrix.config.setup_file }} sdist bdist_wheel
     - name: Publish package
-      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+      # if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
```

Comment:
Temporarily turned off, just it to make the pending 2020.0613 release. Then will revert this line and merge if this pass

Reply:
It worked eventually (from my fork).
Version published in https://pypi.org/project/xcamera/2020.613/
Subsequent tagging should work as expected now